master/buildbot/data/buildrequests.py

```diff
@@ -164,9 +164,9 @@ class EntityType(types.Entity):
 
     @defer.inlineCallbacks
     def generateEvent(self, brids, event):
-        for _id in brids:
+        for id in brids:
```

Comment:
even if this work, I'm not so found of using reserved words as identifiers. This is why I used _id, if you dont like the _ we can call it brid